libs/dh/wholesale/domain/src/calculation.ts

```diff
@@ -27,5 +27,12 @@ export type CalculationGridArea = ResultOf<
   typeof GetCalculationsDocument
 >['calculations'][0]['gridAreas'][0];
 
-export const processTypes = Object.values(ProcessType);
 export const executionStates = Object.values(BatchState);
+export const processTypes = [
+  ProcessType.BalanceFixing,
+  ProcessType.Aggregation,
+  ProcessType.WholesaleFixing,
+  ProcessType.FirstCorrectionSettlement,
+  ProcessType.SecondCorrectionSettlement,
+  ProcessType.ThirdCorrectionSettlement,
+];
```

Comment:
question: was this necessary because you want the processes in specific order? (can see that the enum is ordered differently)